chain_info/Moonbase.yaml

```diff
@@ -0,0 +1,54 @@
+name: Moonbase Alpha
+network_type: testnet
+para_id: 1000
+chain:
+  type: parachain
+  parent: Moonbase Relay (based on Westend)
```

Comment:
This **must** match a valid name relay chain name.

Reply:
It sounds like you miss one file for `Moonbase` (the relaychain).

Reply:
I added a file for Moonbase Relay